exercises/largest-series-product/cases_test.go

```diff
@@ -20,10 +20,10 @@ var tests = []struct {
 	{"73167176531330624919225119674426574742355349194934", 6, 23520, true, ""},
 	{"0000", 2, 0, true, ""},
 	{"99099", 3, 0, true, ""},
-	{"123", 4, -1, false, "span must be smaller than string length"},
+	{"123", 4, -1, false, "string length must be greater than or equal to span"},
 	{"", 0, 1, true, ""},
 	{"123", 0, 1, true, ""},
-	{"", 1, -1, false, "span must be smaller than string length"},
+	{"", 1, -1, false, "string length must be greater than or equal to span"},
 	{"1234a5", 2, -1, false, "digits input must only contain digits"},
-	{"12345", -1, -1, false, "span must be greater than zero"},
+	{"12345", -1, -1, false, "span must be greater than or equal to zero"},
```

Comment:
We could maybe phrase this more concisely as:

> span must not be negative

Reply:
I like that more.